examples/examples/instrumented_error.rs

```diff
@@ -0,0 +1,62 @@
+//! This example demonstrates using the `tracing-error` crate's `SpanTrace` type
+//! to attach a trace context to a custom error type.
+#![deny(rust_2018_idioms)]
+use std::error::Error;
+use std::fmt;
+use tracing_error::{ErrorLayer, Instrument as _, SpanTrace, SpanTraceExt as _};
```

Comment:
seems like this import _might_ be unused... :P
![Screen Shot 2020-02-06 at 11 01 06 AM](https://user-images.githubusercontent.com/2796466/73969308-03100380-48d0-11ea-94ee-368ad5d356d0.png)

Reply:
(also: i think there should eventually be a prelude for sticking the extension traits in, so we don't have to import them as `_`, but we can add that later)